data/json/items/armor/shields.json

```
@@ -161,5 +161,72 @@
     "bashing": 10,
     "encumbrance": 30,
     "techniques": [ "WBLOCK_3" ]
+  },
+  {
+    "id": "shield_welded",
+    "type": "ARMOR",
+    "name": { "str": "welded shield" },
+    "description": "A shield made out of welded sheet metal.  It's a lot heavier and less well-made compared to a proper medieval shield, but it certainly works.",
+    "weight": "8000 g",
+    "volume": "5 L",
+    "price_postapoc": "5 USD",
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel" ],
+    "looks_like": "shield_ballistic",
+    "symbol": "[",
+    "color": "blue",
+    "covers": [ "arm_either", "hand_either" ],
+    "coverage": 90,
+    "encumbrance": 40,
+    "material_thickness": 3,
+    "techniques": [ "WBLOCK_3" ],
```

Consistent with ballistic shield.
```suggestion
    "techniques": [ "WBLOCK_2" ],
```